### ISSUE TITLE
Tiny 7: CHANGELOG?

### ISSUE THREAD

**Issue by Georg-Git** [tinymce/tinymce#9466]
Thanks for version 7! 👍

At first I missed the "master" version on GIT - and was a little bit confused about the new file structure on Git.
But now I'm back on track again and building Tiny 7 without any problems.

However – I am still confused about the correct place of CHANGELOG. 
I was used to find the appropriate file here:
https://github.com/tinymce/tinymce/blob/release/7/modules/tinymce/CHANGELOG.md
Obviously this file is not updated to version 7.

Now I found some newer files here:
https://github.com/tinymce/tinymce/tree/20d9aae8f888f85883d36717b91bbe25ce22c223/.changes/tinymce

Where should I look for the CHANGELOG file in the future?

**Comment by TheSpyder** [comment]
TinyMCE 7 has not been released. Reports of the release are greatly exaggerated. The repository was tagged as 7.0.0 by accident, apologies for the confusion and we have now deleted the tag.

`master` was previously a continuous "current major release" branch, the plan now is to use a new `release/X` for each major. The bleeding-edge `develop` is now `main`.

When we are ready to release TinyMCE 7 the changelog will be updated, and it will be released to NPM.

We have switched to [changie](https://changie.dev/) to avoid constant git conflicts in the changelog file. If you'd like to see what the current list of changes is, the command `changie-merge` will generate the `CHANGELOG.md` files you're used to seeing. We just won't be committing those updates to the repository until release day.

**Comment by Georg-Git** [comment]
Thanks for clarification.

The main confusion was a result of this information:
https://github.com/tinymce/tinymce
"Using an old version of TinyMCE? We recommend you to upgrade to **TinyMCE 7**"

Do You have a rough timeplaning for the new major release 7?

The second quarter of this year?

**Comment by TheSpyder** [comment]
Yes, sorry about that. Previously we would update the readme and publish to NPM on the same day. Docs and the premium release would follow some weeks later.

We are now delaying the NPM release until everything passes QA, which will be less confusing for NPM users, but I can see how it's more confusing for anyone using the git repo directly. We need some way to mark the `release/7` branch as "draft”.

We will take this feedback and evaluate whether we can do better for the eventual TinyMCE 8 release.

I don't want to give a hard ETA but I am hopeful it will be before the end of Q1.

**Comment by TheSpyder** [comment]
Yes, part of the new approach is we don't update the changelog file until release day. The place to keep an eye on is the release branch:
https://github.com/tinymce/tinymce/blob/release/7/modules/tinymce/CHANGELOG.md

`main` will be used for development of the next minor (or major) release, the `release/7` branch is where all the 7-specific changes will be made.

We will consider TinyMCE 7 to be released when it hits NPM, which is still 6.8.3 - perhaps we should add one of those NPM status tags to the readme 😅
https://www.npmjs.com/package/tinymce

**Comment by TheSpyder** [comment]
TinyMCE 7 has now been released!
https://www.npmjs.com/package/tinymce

**Comment by Georg-Git** [comment]
Thanks again - now for the official release of TinyMCE 7 🤣

Building it with windows yarn - no problems.
Rebuild my project with TinyMCE 7 - no problems.

So - You made Your job 👍

Of course I had noticed the warning in debugger console of my browser:
https://www.tiny.cloud/docs/tinymce/latest/license-key/

No problem.
